terraform/aws/analytical-platform/oidc/configuration/assumable-roles.json

```diff
@@ -176,12 +180,28 @@
   "modernisation-platform-lake-formation-share": {
     "account": "analytical-platform-data-production",
     "stateConfig": [],
-    "repositories": [
-      "ministryofjustice/modernisation-platform-environments"
-    ],
+    "repositories": ["ministryofjustice/modernisation-platform-environments"],
     "targets": [],
     "stateLockingDetails": [],
     "ssmParameterConfig": [],
     "lakeFormationSharePolicy": true
+  },
+  "create-a-derived-table-infrastructure": {
```

Comment:
Can you double-check this doesn't use a state? There's a mention of pulumi in the docs so I'd want to be careful.

Reply:
Both repositories are Pulumi Python and use the bucket `s3://data-engineering-pulumi.analytics.justice.gov.uk/.pulumi/`  and within the `stacks` folder are the files `de-database-access.json` and `create-a-derived-table-infra.json` which i have inferred are the state files for Pulumi. The stateConfig has been updated accordingly.